PubHubs/Issues/account/description.xml

```diff
@@ -0,0 +1,55 @@
+<IssueSpecification version="4">
+	<Name>
+		<en>Demo Account</en>
+		<nl>Demo Account</nl>
+	</Name>
+	<ShortName>
+		<en>Demo Account</en>
+		<nl>Demo Account</nl>
+	</ShortName>
+	<SchemeManager>irma-demo</SchemeManager>
+	<IssuerID>PubHubs</IssuerID>
+	<CredentialID>account</CredentialID>
+	<Description>
+		<en>Your PubHubs account</en>
+		<nl>Je PubHubs account</nl>
+	</Description>
+	<ShouldBeSingleton>false</ShouldBeSingleton>
+	<IssueURL>
+		<en>https://privacybydesign.foundation/attribute-index/en/irma-demo.PubHubs.account.html</en>
+		<nl>https://privacybydesign.foundation/attribute-index/nl/irma-demo.PubHubs.account.html</nl>
+	</IssueURL>
+
+	<Attributes>
+		<Attribute id="id">
+			<Name>
+				<en>Identification number</en>
+				<nl>Identificatie nummer</nl>
+			</Name>
+			<Description>
+				<en>Your unique identification number</en>
+				<nl>Je unique Identificatie nummer</nl>
```

Comment:
hoofdletter i vervangen door kleine i
identificatienummer is in het nederlands in principe 1 woord